style.css

```diff
@@ -0,0 +1,1078 @@
+  font-size: 14px;
+  margin-bottom: 10px;
+  color: #272a31;
+}
+
+.mail_to_list {
+  font-weight: 600;
+}
+
+.mail_to_content {
+  color: #272a31ad;
+  font-size: 13px;
+}
+
+.second_header_section {
+  padding: 50px;
+  text-align: center;
+}
+
+.second_header_section_title {
+  font-weight: 500;
+  color: #272a31;
+}
+
+.seperatorV {
+  background-color: #ec5242;
+  height: 1px;
+  width: 30px;
+  margin-left: 600px;
+  margin-top: 10px;
+  margin-bottom: 30px;
+}
+
+.second_header_section_content {
+  color: #272a31;
+  line-height: 20px;
+  font-size: 13px;
+  width: 60%;
+  margin-left: 250px;
+}
+
+.dev_logo_container {
+  border: 1px solid #d3d3d3;
+  padding: 30px;
+  width: 30%;
+  margin-left: 450px;
+  margin-top: 40px;
+  margin-bottom: 20px;
+}
+
+.seperatorVII {
+  background-color: #d3d3d3;
+  height: 1px;
+  width: 100%;
+}
+
+.third_section_header_title {
+  margin-top: 40px;
+}
+
+.third_section_title {
+  font-size: 16pt;
+  font-weight: 500;
+  color: #272a31;
+}
+
+.seperatorX {
+  background-color: #ec5242;
+  width: 30px;
+  margin-left: 600px;
+  height: 1px;
+  margin-top: 10px;
+  margin-bottom: 20px;
+}
+
+.senior_devs {
+  display: flex;
+  flex-direction: row;
+  width: 50%;
+  margin-left: 320px;
+}
+
+.senior_dev_list {
+  width: 100%;
+  padding: 40px;
+  margin: 10px;
+}
+
+.senior_dev_listI {
+  background-image: url(./images/Historical_Devs.jpg);
+  opacity: 90%;
+  background-position: center;
+  background-size: cover;
+}
+
+.Dev_container {
+  background-color: #272a31;
+  opacity: 90%;
+  padding: 20px;
+}
+
+.Development_year {
+  color: #d3d3d3;
+}
+
+.Development_content {
+  color: #d3d3d3;
+  font-size: 13px;
+}
+
+.senior_dev_listII {
+  background-image: url(./images/second_Dev_history.jpg);
+  background-position: center;
+  background-size: cover;
+}
+
+.third_section_title_content_item {
+  font-size: 13px;
+  margin-bottom: 40px;
+}
+
+.footerII {
+  display: flex;
+  padding: 60px;
+  background-color: #272a31;
+  color: #d3d3d3;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.social_mid {
+  width: 15px;
+  height: 18px;
+}
+
+.navbar {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  margin-top: 20px;
+  background-color: #fff;
+}
+
+.logo_navbar {
+  width: 200px;
+  height: 50px;
+  margin-left: 150px;
+}
+
+.logo_navbar:hover {
+  transform: rotateX(360deg);
+  transition: all 2s;
+}
+
+/* DESKTOP NAVBAR */
+.navbar_wrapper {
+  display: flex;
+}
+
+.mobile_nav {
+  display: none;
+}
+
+.destop_content_list {
+  list-style-type: none;
+  margin-top: 16px;
+  padding-left: 40px;
+  font-size: 15px;
+}
+
+.container_list {
+  display: flex;
+  padding-left: 220px;
+}
+
+/* HEADER */
+.header {
+  position: fixed;
+  width: 100%;
+  align-items: center;
+  background-color: #272a31;
+  justify-content: right;
+  height: 20px;
+  font-family: sans-serif;
+}
+
+.second_header_container {
+  padding: 150px;
+  background-image: url(./images/first_bg.jpg);
```

Comment:
- [ ] [OPTIONAL]: Great job adding this image as a background. You should consider adding a `linear-gradient`, a white one maybe, to help improve readability on mobile version (the bright colors of the image sometimes makes it hard to read the text). You can read more on how apply a filter to a background image [here](https://css-tricks.com/apply-a-filter-to-a-background-image/).